### ISSUE TITLE
Remove output tensors 1 - 4 from FusedBatchNorm

### ISSUE THREAD

**Issue by santoshchilkunda** [tensorflow/tensorflow#23143]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 1.10.0
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
Downloaded frozen inference model MobileNet_v1_1.0_224 from https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.md

FusedBatchNorm has five outputs. 1-4 seem to be related to training. How do I remove them?

I tried the following code:

			if op.type == 'FusedBatchNorm':
				new_node = node_def_pb2.NodeDef()
				new_node.name = op.node_def.name
				new_node.op = op.node_def.op
				for i in op.node_def.input:
					new_node.input.extend([i])
				for a in op.node_def.attr:
					new_node.attr[a].CopyFrom(op.node_def.attr[a])
				output_graph.node.extend([new_node])

			else:
				new_node = node_def_pb2.NodeDef()
				new_node.CopyFrom(op.node_def)
				output_graph.node.extend([new_node])

It still has the output tensors 1 - 4.

Can this feature be added to Graph Transform tool?

**Will this change the current api? How?**
It won't. It's a new transformation in Graph Transformation tool.

**Who will benefit with this feature?**
Mobilenet users

**Any Other info.**

**Comment by santoshchilkunda** [comment]
Have I written custom code? Yes
		if op.type == 'FusedBatchNorm':
			new_node = node_def_pb2.NodeDef()
			new_node.name = op.node_def.name
			new_node.op = op.node_def.op
			for i in op.node_def.input:
				new_node.input.extend([i])
			for a in op.node_def.attr:
				new_node.attr[a].CopyFrom(op.node_def.attr[a])
			output_graph.node.extend([new_node])

		else:
			new_node = node_def_pb2.NodeDef()
			new_node.CopyFrom(op.node_def)
			output_graph.node.extend([new_node])

OS Platform and Distribution: Ubuntu 16.04

TensorFlow installed from: pip

Bazel version: N/A

CUDA/cuDNN version: N/A 

GPU model and memory: No GPU support

Exact command to reproduce: N/A

Mobile device: N/A

**Comment by wt-huang** [comment]
@santoshchilkunda This is a valid point though Batch Normalization can be used for inference as well. 
You can delete the 4 extra outputs in FusedBatchNorm leaving only output y which is a 4D tensor for output data. Try to use onnx tool to remove those extra outputs.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @wt-huang: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by gowthamkpr** [comment]
@wt-huang You can take a look at the solution provided [here](https://stackoverflow.com/questions/59270063/batch-normalization-at-inference-time-in-tensorflow). Look at the comments of the answer for the solution.  Also, if the issue still persists, please post this issue in [tensorflow/models](https://github.com/tensorflow/models/issues) as anything related to the models in that repo are discussed there. Thanks!!
